src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <label class="bar">
+        <input
+          class="input"
+          type="search"
+          name="bigBar"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+        <img
+          class="icon"
+          src="images/Search.svg"
+          alt="big search icon"
+        >
```

Comment:
Better to use background img for decorative img

Reply:
> Better to use background img for decorative img

better, but not necessary?...